### ISSUE TITLE
Fix TypeError for spaceless condition in media-feature-name-value-allowed-list

### ISSUE THREAD

**Issue by siarheiyelin** [stylelint/stylelint#5577]
> Clearly describe the bug

Stylelint throws error when "media-feature-name-value-allowed-list" rule is defined for the following SCSS code:
SCSS code looks like this:
```scss
// This code causes error (even though it's correct SCSS code). Important note: there is no space between "and" and "("
        @media only screen and(min-width: 640px) {
            display: none;
        }
// Next code is OK, notice that space is aded between "and" and "("
        @media only screen and (min-width: 640px) {
            display: none;
        }
```

Error looks like this:
```
(node:1652) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'value' of undefined
    at base.scss:995:13
    at node_modules\stylelint\lib\rules\media-feature-name-value-allowed-list\index.js:53:42
    at node_modules\postcss-media-query-parser\dist\nodes\Container.js:62:33)
    at node_modules\postcss-media-query-parser\dist\nodes\Container.js:65:28)
    at node_modules\stylelint\lib\rules\media-feature-name-value-allowed-list\index.js:34:31
    at node_modules\postcss-scss\node_modules\postcss\lib\container.js:307:18
    at node_modules\postcss-scss\node_modules\postcss\lib\container.js:139:18
```

> Which rule, if any, is the bug related to?

It's related to "media-feature-name-value-allowed-list"

> What stylelint configuration is needed to reproduce the bug?
```js
{
  "rules": {
        'media-feature-name-value-allowed-list': [
            {
                'max-width': ['640px'],
            },
        ],
  }
}
```

> Which version of stylelint are you using?

"stylelint": "13.13.1"

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

It doesn't depend on how it's run. I was able to reproduce it by running via CLI and by running via Webstorm.

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

No

> What did you expect to happen?

Rule doesn't throw exception

> What actually happened (e.g. what warnings or errors did you get)?

Rule throws exception
